.github/actions/set-up-flutter-and-dart/flutter_ref-beta

```diff
@@ -0,0 +1 @@
+465e421f53c05f8f33f83a243c10c55f9f6efbc2
```

Comment:
How are these hashes kept up to date?

Reply:
I realized we don't even need flutter.

I removed the `set-up-dart-and-flutter` action altogether: https://github.com/atsign-foundation/at_services/pull/52/commits/bacdf8e0409c995470ebce4918e0060082d674ea https://github.com/atsign-foundation/at_services/pull/52/commits/cd1588af31c2501b904ec6f691717a1a7e86a559

Instead we are now using the `dart-lang/setup-dart` action